README.md

```diff
@@ -181,3 +181,9 @@ used in the creation of [configure](configure) from
 [configure.ac](configure.ac), and are provided as a courtesy to
 developers. You are free to substitute them with other autoconf macros
 that provide equivalent functionality.
+
+BOUT++ links by default with gettext and fftw, both are licensed as
```

Comment:
Could we make this a bit more generic rather than specifying some of the GPL licensed things we can use so that we don't have to remember to update this as and when we add more links to external GPL licensed things? This would help us avoid accidentally misleading by omission.

Reply:
I now removed the references to the libraries :+1: